Dockerfile

```diff
@@ -6,13 +6,27 @@ ARG KUBERNETES_VERSION=
 
 RUN set -x && \
     apt-get update && \
-    apt-get install -y jq curl && \
+    apt-get install -y jq curl git && \
```

Comment:
Surprisingly (?) `go get` requires git.

Reply:
By using multi-stage-builds with golang:1.10, we don't need to install git.